rotkehlchen/globaldb/asset_updates/manager.py

```diff
@@ -41,7 +41,7 @@
 ASSET_COLLECTIONS_MAPPINGS_UPDATES_URL: Final = 'https://raw.githubusercontent.com/rotki/assets/{branch}/updates/{version}/asset_collections_mappings_updates.sql'
 FIRST_VERSION_WITH_COLLECTIONS: Final = 16
 FIRST_GLOBAL_DB_VERSION_WITH_COLLECTIONS: Final = 4
-FIRST_VERSION_WITH_SOLANA_TOKENS: Final = 37
+FIRST_GLOBAL_DB_VERSION_WITH_SOLANA_TOKENS: Final = 13
```

Comment:
This was checking the assets version before but now it's again back to the global DB version. Why? Before it would break for people having 1.39.1? No that can't be as this is code for 1.40.0.

Reply:
the problem happens when users on app version 1.39.1 (assets v36, global db v12) upgrade to 1.40.0 (assets v37, global db v13). 

we copy their data to a temporary database, but `solana_tokens` doesn't get copied because i incorrectly used the assets version to determine which tables to include. 

so, when we try to merge the updated data back, it fails because we're inserting solana tokens data into a database without that table structure.